tests/lib/in_memory/database.py

```diff
@@ -344,3 +302,58 @@ def parse_value(value):
 def nulls_first_order(key):
     # Usable as a key function to `sorted()` which sorts NULLs first
     return (0 if key is None else 1, key)
+
+
+def apply_function_to_columns(fn, *columns):
```

Comment:
When we handle a `Case` node, we're not passing in columns.  Could we call this `apply_function`?

Reply:
Fine by me!